### ISSUE TITLE
Open Documentation

### ISSUE THREAD

**Issue by catdisc** [DavidGriffith/inform6unix#8]
I see now that you've removed the non-free Inform 6 Designer's Manual. I wanted to mention that there is openly licensed documentation available, in the form of the Inform Beginner's Guide, as the authors were willing to re-license it to something compatible with the Debian Free Software Guidelines (DFSG). (The manual previously disallowed profit and modification, both of which conflicted with the DFSG.) While not a replacement for the Inform 6 Designer's Manual it is a start to something. I'm sending it your way in case you're interested in adding it. Notice the updated page 2.

I'm told that it was originally done in Adobe FrameMaker, which is proprietary, and that the source files have been lost; all that remains  is the PDF. But since the PDF has text that can be copied out my hope is that, someday, it can be converted back into something more editable than a PDF.

[IBG.pdf](https://github.com/DavidGriffith/inform6unix/files/190741/IBG.pdf)

**Comment by DavidGriffith** [comment]
Somehow inform-fiction.org is still serving up the second edition.  I'll contact Roger Firth and see if he'd be interested in accepting changes to make a fourth edition.

**Comment by zondo** [comment]
Might I suggest using [Sphinx](http://sphinx-doc.org) for the converted documentation format?  It's more contributor-friendly than LaTeX and makes really great-looking online docs.  It's such a nice way to produce documentation that I'll even volunteer to do the main conversion.

**Comment by DavidGriffith** [comment]
Sphinx looks nice.  I just now emailed Roger Firth to see how he feels about this effort.

**Comment by DavidGriffith** [comment]
I just now got Roger's blessing to do this.  We're clear to produce a fourth edition.  I'm particularly keen on getting it updated to cover changes up to Compiler 6.33 and Library 6/12.

**Comment by zondo** [comment]
Ok, if you're happy with Sphinx/reStructuredText format, just create a repo under there and give me commit access; I'll make a start on a conversion.

**Comment by DavidGriffith** [comment]
done

**Comment by zondo** [comment]
Great.  I'll get started on it some time this week.

**Comment by DavidGriffith** [comment]
Did you get the source files I emailed you?

**Comment by zondo** [comment]
Yes I did, thanks.  Very kind of Mr Firth to make them available.  Unfortunately, there doesn't seem to be anything that'll read FrameMaker files except FrameMaker itself.  But the images and screenshots are separate, and will be very useful.  Together with text dumped from the PDF file, that should be enough to get it done.

I'm just getting the initial repo in a fit state for publication; conversion is done up to the end of chapter 1 so far.  Should have something ready for upload this evening.  And then you can test how well I've written my build instructions :-)

**Comment by DavidGriffith** [comment]
Will Sphinx be able to emit something that can be sent to have made into a bound book?

**Comment by zondo** [comment]
Yes -- Sphinx has a big bunch of output formats, including LaTeX for producing PDFs.  Check out the Sphinx link a few comments back for the details.

**Comment by zondo** [comment]
Ok, after a bit of hg-git annoyance, the initial repo is uploaded.  Give it a go!
